### PR TITLE
SkystarsH743HD target

### PR DESCRIPTION
**Pull request by darsanjp**
Added SkystarsH743HD target.

### CODE REVIEW

File: src/main/target/SKYSTARSH743HD/config.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of Cleanflight.
```
Comment (DzikuVx) [inline src/main/target/SKYSTARSH743HD/config.c]: Am, not, it's not Cleanflight, so please update the project name

File: src/main/target/SKYSTARSH743HD/config.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#include "fc/fc_msp_box.h"
+#include "fc/config.h"
+
+#include "io/piniobox.h"
+
+void targetConfiguration(void)
+{
+    pinioBoxConfigMutable()->permanentId[0] = BOX_PERMANENT_ID_USER1;
+    pinioBoxConfigMutable()->permanentId[1] = BOX_PERMANENT_ID_USER2;
+   // beeperConfigMutable()->pwmMode = true;
```
Comment (DzikuVx) [inline src/main/target/SKYSTARSH743HD/config.c]: Not needed

File: src/main/target/SKYSTARSH743HD/target.c [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    DEF_TIM(TIM4, CH2, PD13, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 3),   // S4
+
+    DEF_TIM(TIM5, CH1, PA0, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 4),   // S5  
+    DEF_TIM(TIM5, CH2, PA1, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 5),   // S6
+    DEF_TIM(TIM5, CH3, PA2, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 6),   // S7
+    DEF_TIM(TIM5, CH4, PA3, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 7),   // S8
+
+    
+    //DEF_TIM(TIM4, CH3, PD14, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),   // S9
+    //DEF_TIM(TIM4, CH4, PD15, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),   // S10 DMA_NONE
```
Comment (DzikuVx) [inline src/main/target/SKYSTARSH743HD/target.c]: Not needed, so remove

**Review by DzikuVx — CHANGES_REQUESTED** [review]
Please read https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

As this is a new target, this PR will not be merged if all the conditions from this document are not met

File: src/main/target/SKYSTARSH743HD/config.c (hunk 1 above)
Comment (darsanjp) [reply src/main/target/SKYSTARSH743HD/config.c]: > Am, not, it's not Cleanflight, so please update the project name

Corrected the project name .

File: src/main/target/SKYSTARSH743HD/config.c (hunk 2 above)
Comment (darsanjp) [reply src/main/target/SKYSTARSH743HD/config.c]: Removed the commented out code.

File: src/main/target/SKYSTARSH743HD/target.c (hunk 3 above)
Comment (darsanjp) [reply src/main/target/SKYSTARSH743HD/target.c]: Removed the commented out code.

**Comment by darsanjp** [comment]
> Please read https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
> 
> As this is a new target, this PR will not be merged if all the conditions from this document are not met

I bought this board for a cinewoop and later found that I cant use INAV as INAV target was not available. Target for this board exists in Betaflight. I spend some time with the betaflight config file and was then able to compile an INAV target. Thanks to your video on INAV custom targets. Currently using on my quad with gps and matek lidar sensor. Working like charm. Thought of sharing it with the community. I have read the new hardware policy. Unfortunately I dont have a spare board to send to the developers. But I am willing to spend my time for development for this project.

**Comment by erstec** [comment]
> > Please read https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
> > As this is a new target, this PR will not be merged if all the conditions from this document are not met
> 
> I bought this board for a cinewoop and later found that I cant use INAV as INAV target was not available. Target for this board exists in Betaflight. I spend some time with the betaflight config file and was then able to compile an INAV target. Thanks to your video on INAV custom targets. Currently using on my quad with gps and matek lidar sensor. Working like charm. Thought of sharing it with the community. I have read the new hardware policy. Unfortunately I dont have a spare board to send to the developers. But I am willing to spend my time for development for this project.

You don't need to have spare boards and do anything about this at all. It is 100% manufacturer responsibility to do that.

**Comment by DzikuVx** [comment]
Thanks @darsanjp 
Replaced by #8954

**Pull request closed without merging**